boilerplate/openshift/golang-osd-operator/rvmo-bundle.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+# TODO: some syntax error here
+REPOSITORY=${REPOSITORY:"git@github.com:openshift/managed-release-bundle.git"}
+BRANCH=${BRANCH:main}
```

Comment:
If the idea is to set these variables if they are unset, try
```suggestion
REPOSITORY=${REPOSITORY:-"git@github.com:openshift/managed-release-bundle.git"}
BRANCH=${BRANCH:-main}
```

Reply:
thank you, Eric! :)